index.d.ts

```diff
@@ -0,0 +1,104 @@
+// TypeScript Version: 3.7
+
+import {Test} from 'unist-util-is'
+import * as unist from 'unist'
+
+export {}
+
+type Null = undefined | null
+
+declare namespace messageControl {
+  export {}
+  /** a comment marker */
```

Comment:
Try to keep using same description from readme.

```ts
/**
 * A comment marker.
 */
interface Marker<N extends unist.Node>{
```

Reply:
Is the issue here the missing punctuation?